src/scalar.jl

```diff
@@ -218,6 +356,52 @@ const asℝ = as(Real, -∞, ∞)
 
 const as_real = asℝ
 
+# Fallback method: print all transforms in order
+function Base.show(io::IO, ct::CompositeScalarTransform)
+    str = string(ct.transforms[1])
+    for ti in ct.transforms[begin+1:end]
+        str *= " ∘ "*string(ti)
+    end
+    print(io, str)
+end
```

Comment:
It's inefficient to create a string only to print it afterwards. But here it's even simpler, this is just
```suggestion
Base.show(io::IO, ct::CompositeScalarTransform) = join(io, ct.transforms, " ∘ ")
```

Reply:
Good point!